infra/create-app/int.tfvars

```diff
@@ -0,0 +1,8 @@
+domain_name = "int.octopus.ac"
```

Comment:
If i remember right, we're not suppose to include these values in the repo? Treating them like .env

Reply:
Ok cool, thanks for clarifying 👍🏼